templates/flutter/lib/services/service.dart.twig

```diff
@@ -1,14 +1,11 @@
 part of {{ language.params.packageName }};
-{% macro parameter(parameter) %}{% if parameter.required %}required {{ parameter.type | typeName }}{% else %}{{ parameter.type | typeName }}?{% endif %} {{ parameter.name | caseCamel | overrideIdentifier }}{% endmacro %}
-{% macro method_parameters(parameters, consumes) %}
-{% if parameters|length > 0 %}{{ '{' }}{% for parameter in parameters %}{{ _self.parameter(parameter) }}{% if not loop.last %}, {% endif %}{% endfor %}{% if 'multipart/form-data' in consumes %}, Function(UploadProgress)? onProgress{% endif %}{{ '}' }}{% endif %}
-{% endmacro %}
+{% import 'flutter/base/utils.twig' as utils %}
```

Comment:
we did not export this to utils, as this macro is used only in this file

Reply:
Right, but since I needed this in `templates/flutter/base/requests/location.twig`, I thought it would be better to move it to utils.